js/views/messagesitem.js

```diff
@@ -98,6 +98,8 @@ define(function(require) {
 		openMessage: function(event) {
 			event.stopPropagation();
 			$('#mail-message').removeClass('hidden-mobile');
+			// make sure message is marked as read when clicked on it
+			Radio.ui.trigger('messagesview:messageflag:set', this.model.id, 'unseen', false);
```

Comment:
Updated my branch to your changes: https://github.com/nextcloud/mail/pull/326/commits/2337f74ef361da9d25ff0d4e305a0e24815d0f70

Following Linus's law *given enough eyeballs, all bugs are shallow* it would be great if you could verify I resolved the conflicts correctly :-)